package.json

```diff
@@ -16,7 +16,8 @@
     "react-is": "16.13.1",
     "winston@3.8.2": "patch:winston@npm:3.8.2#.yarn/patches/winston-npm-3.8.2-2035e9cac4.patch",
     "ramda@0.28.0": "patch:ramda@npm:0.28.0#.yarn/patches/ramda-npm-0.28.0-6a5fe8f6cc.patch",
-    "postcss": "8.4.31"
+    "postcss": "8.4.31",
+    "@types/react": "18.2.25"
```

Comment:
Quick question: why did we have to pin the version of @types/react?

Reply:
Seems the update to `@types/react-helmet` introduced a conflict with the version of `@types/react` its using: `https://github.com/nfl/react-helmet/issues/646#issuecomment-1742976025`

`react-helmet` is no longer supported and is something we should _probably_ move away from. `react-helmet-async` is the alternative I believe, or we can hold off and migrate to Next.js which has built-in methods like `next/head` and `next/script` that are better.

Reply:
I believe we have a ticket in the backlog to move to `react-helmet-async`

Reply:
Worryingly though, `react-helmet-async` doesn't look to be the best supported either: https://github.com/staylor/react-helmet-async + `https://github.com/staylor/react-helmet-async/issues/194`